get_ventas_lista.md

```
@@ -1 +1,135 @@
-Prueba
+### Descripción:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)